aries/aries_vcx_core/src/utils/mod.rs

```diff
@@ -1,3 +1,3 @@
 pub mod async_fn_iterator;
-pub(crate) mod constants;
```

Comment:
The `constants` are only used in tests, not mean to be part of public interface. Move the constants if possible, or copy if necessary

Reply:
Changed.